public/assets/temporary/custom-style.css

```diff
@@ -67,5 +67,5 @@ h5 {
 .muc-divider {
     align-self: stretch;
     height: 0;
-    border: 1px var(--color-neutrals-blue) solid;
+    border-bottom: 1px var(--color-neutrals-blue) solid;
 }
```

Comment:
We should outsource this css-class into a css-file inside the muc-divider folder or into the component itself.

If this css class isn't used by any other component the later way would be prefered.

Reply:
It is used by two more components (MucCard, MucIntro).
As our design coordinator mentioned, the height of the divider should be 1px in all these instances.

Reply:
With [this](https://github.com/it-at-m/muc-patternlab-vue/pull/282/commits/dd7317cef8da72df81402ae7fd6af4c1c00b04cd) commit, they do not use it any more. If we already have the component, we can use in other components too.

I only found those two occurrences.